### ISSUE TITLE
Regex rename class is broken

### ISSUE THREAD

**Issue by Ducasse** [guillep/pharo-core#13766]
```
className: aName newName: aNewName
	className := aName asSymbol.
	class := self model classNamed: className.
	newName := aNewName asSymbol
```
aName is a class.

```
className: aName newName: aNewName
	className := aName name asSymbol.
	class := self model classNamed: className.
	newName := aNewName asSymbol
```

**Comment by balsa-sarenac** [comment]
The command uses the `RBRenameClassTransformation` instead of `RBRenameClassRefactoring`. The command passes the class name to the refactoring/transformation, you can check `SycRenameClassCommand>>#execute`.

I've tested renaming it from multiple places, and every time it worked fine. I've also changed it to use the refactoring and it worked OK as well. Was there a specific scenario where it didn't work for you?

**Comment by Ducasse** [comment]
I was executing the code snippet in the class RegexClassRename so this is probably the culprit

**Comment by balsa-sarenac** [comment]
Yes, it was. I've added the fix: #13770

**Comment by balsa-sarenac** [comment]
@Ducasse can you rename the issue so it says the regex class rename was broken?
